erlang/question.erl

```diff
@@ -12,5 +16,7 @@ question(Prompt, Valid) ->
     end.
 
 main(_) -> 
-    Result = question("Foo", ["Bar", "Baz"]),
-    io:format("~s~n", [Result]).
+    A = question("Foo", ["Bar", "Baz"]),
+    io:format("~s~n", [A]),
+    B = question("Foo"),
+    io:format("~s~n", [B]).
```

Comment:
remove this line and the two above it for consistency with other implementations